poc_iot_verifier/src/poc.rs

```diff
@@ -112,8 +109,11 @@ impl Poc {
         let beacon_received_time = self.beacon_report.received_timestamp;
         if beacon_received_time < self.entropy_start || beacon_received_time > self.entropy_end {
             tracing::debug!(
-                "beacon verification failed, reason: {:?}",
-                InvalidReason::BadEntropy
+                "beacon verification failed, reason: {:?}. beacon_received_time: {:?}, entropy_start_time: {:?}, entropy_end_time: {:?}",
+                InvalidReason::BadEntropy,
+                beacon_received_time,
```

Comment:
you know you can inline these single variable names right? i.e. in the format string you can `{beacon_received_time:?}` and remove this positional argument.

Reply:
yip am aware, switch between the two for no valid reason